docs/index.rst

```diff
@@ -27,4 +27,5 @@ Introductory topics
     Configuration
     Logging
     Connecting_to_web_UIs
+    Configuring_PuTTY_on_Windows 
```

Comment:
don't include it here, it should be linked from the the "Connecting to web UIs" page?